main/admin-panel/types.ts

```diff
@@ -0,0 +1,39 @@
+export interface Job {
+  id: string;
+  title: string;
+  description: string;
+  is_active: boolean;
+  date_created: Date;
+  date_updated: Date;
+}
+
+export interface Testimonial {
+  id: string;
+  author: string;
+  position: string;
+  testimonial: string;
+  tech_stack: string[];
+  picture_url: {
+    rawFile: File;
+    src: string;
+    title: string;
+  };
+  is_approved: boolean;
+}
+
+export interface TeamMember {
+  id: string;
+  name: string;
+  position: string;
+  bio: string;
+  experience_years: number; 
+  picture_url: {
+    rawFile: File;
+    src: string;
+    title: string;
+  };
+  tech_stack: string[];
+  is_active: boolean;
+  date_created: Date;
+  date_updated: Date;
+}
```

Comment:
Consider using `@scytix/common` module where you can share code between backend and front end.